resources/js/Shared/Widgets/HomeOfficeList.vue

```diff
@@ -26,11 +29,28 @@
           </div>
         </li>
       </ul>
+      <EmptyState
+        v-else
+        :show-description="false"
+      >
+        <template #head>
+          <HomeCityIcon
+            class="flex justify-center"
+            size="48"
+          />
+        </template>
+        <template #title>
+          Nikt nie pracuje z domu
```

Comment:
```suggestion
          Nikt nie pracuje zdalnie
```

You don't need to sit at home during remote work. ;)